sdk/sdk-common/src/protocols/interfaces/AaveV3PoolId.ts

```diff
@@ -0,0 +1,17 @@
+import {ChainInfo} from "~sdk-common/common";
+import {IPoolId, ProtocolName} from "~sdk-common/protocols";
+import { EmodeType } from '../enums/EmodeType'
+
+
+// TODO: temporary interface so FE can create this data types without talking to a service
+export interface AaveV3PoolId extends IPoolId {
+    protocol: {
```

Comment:
This should be `IProtocol` instead

Reply:
It's a more narrow form of IProtocol? Conforms to IProtocol but only allow AAVEv3 as protocol name

Reply:
Then I guess it should be `AAVEv3Protocol`, which does not exist at this moment. I guess the plugin should provide those specialized types:
```
interface AAVEv3Protocol extends IProtocol {
  name: ProtocolName.AAVEv3
  
  // whatever other specific data here
}